style.css

```diff
@@ -0,0 +1,64 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 25px;
+}
+
+.logo {
+  flex-grow: 1;
+  flex-shrink: 5;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  padding: 25px;
+  background-image: url("./images/background.svg");
+  background-size: 100vw;
+  background-repeat: no-repeat;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  margin: 12px 0;
+  padding-top: 50px;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
```

Comment:
- [ ] Kindly change the font for this paragraph to make it like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/154376561-c952cb8e-8641-4a61-a44e-94d61bdf419f.png) | ![image](https://user-images.githubusercontent.com/35535130/154376611-3ffba849-aea1-4451-941e-57104eb3855c.png) |